### PR TITLE
[ADD] Add web_hideleftmenu module

### PR DESCRIPTION
**Pull request by aaltinisik**
Updated web_hideleftmenu module of vauxoo to V8.0

This module should be included in OCA repo.

[Review on Reviewable](https://reviewable.io/reviews/oca/web/294)

### CODE REVIEW

File: web_hideleftmenu/README.rst [hunk 1]
```diff
@@ -0,0 +1,81 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
```
Comment (elicoidal) [inline web_hideleftmenu/README.rst]: Thanks for your contribution to the OCA. 
- Please follow the guidelines from:
  https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md
- Please follow the OCA README template:
  https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst
  We recommend to include a screenshot as well to help new comers (nice to have)

File: web_hideleftmenu/README.rst [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+Web Hide Left Menu
```
Comment (elicoidal) [inline web_hideleftmenu/README.rst]: Hide left Menu in Web interface

File: web_hideleftmenu/README.rst [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+==============
```
Comment (elicoidal) [inline web_hideleftmenu/README.rst]: lines need to be until the end of the sentence (the one before and after)

File: web_hideleftmenu/README.rst [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+This module adds a menu item after logout link to hide or show left menu
```
Comment (elicoidal) [inline web_hideleftmenu/README.rst]: periods at the end of the sentences

File: web_hideleftmenu/README.rst [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+It is usefull on small screens or tree views with many columns
```
Comment (elicoidal) [inline web_hideleftmenu/README.rst]: useful

File: web_hideleftmenu/README.rst [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+
+To use this module, you need to:
+
+* Click on new menu item "Hide/Show Menu"on dropdown menu after logout menu item.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/{repo_id}/{branch}
+
+.. repo_id is available in https://github.com/OCA/maintainer-tools/blob/master/tools/repos_with_ids.txt
```
Comment (elicoidal) [inline web_hideleftmenu/README.rst]: remove

File: web_hideleftmenu/README.rst [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+.. branch is "8.0" for example
+
+Known issues / Roadmap
```
Comment (elicoidal) [inline web_hideleftmenu/README.rst]: remove empty section

File: web_hideleftmenu/README.rst [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+======================
+
+* ...
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/{project_repo}/issues>`_. In case of trouble, please
```
Comment (elicoidal) [inline web_hideleftmenu/README.rst]: Please adapt the 2 links according to the module.

File: web_hideleftmenu/README.rst [hunk 9]
```diff
@@ -0,0 +1,81 @@
...
+
+
+Images
+------
+
+
+Contributors
+------------
+
+* Vauxoo
```
Comment (elicoidal) [inline web_hideleftmenu/README.rst]: any email?
as reminder, if this is a company it should in Author section. If individual, put it in this contributor section.

File: web_hideleftmenu/__openerp__.py [hunk 10]
```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
```
Comment (elicoidal) [inline web_hideleftmenu/__openerp__.py:1]: Please use simple headers in all py files following OCA templates
https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3

File: web_hideleftmenu/__openerp__.py [hunk 11]
```diff
@@ -0,0 +1,59 @@
...
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    "name": "Web Hide Left Menu",
+    "summary": """
```
Comment (elicoidal) [inline web_hideleftmenu/__openerp__.py]: remove

File: web_hideleftmenu/__openerp__.py [hunk 12]
```diff
@@ -0,0 +1,59 @@
...
+
+    """,
+    "version": "8.0.1.0.0",
+    "category": "Hidden",
+    "website": "https://odoo-community.org/",
+    "author": "Vauxoo, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "external_dependencies": {
```
Comment (elicoidal) [inline web_hideleftmenu/__openerp__.py]: remove empty key

**Comment by pedrobaeza** [comment]
I have tried on runbot and I'm not able to see the menu. Can you point me the problem?

File: web_hideleftmenu/README.rst (hunk 3 above)
Comment (aaltinisik) [inline web_hideleftmenu/README.rst]: done, thanks

File: web_hideleftmenu/README.rst (hunk 4 above)
Comment (aaltinisik) [inline web_hideleftmenu/README.rst]: thanks

File: web_hideleftmenu/README.rst (hunk 7 above)
Comment (aaltinisik) [inline web_hideleftmenu/README.rst]: done, thanks

File: web_hideleftmenu/README.rst (hunk 8 above)
Comment (aaltinisik) [inline web_hideleftmenu/README.rst]: added links

File: web_hideleftmenu/__openerp__.py (hunk 10 above)
Comment (aaltinisik) [inline web_hideleftmenu/__openerp__.py:1]: done

File: web_hideleftmenu/__openerp__.py (hunk 12 above)
Comment (aaltinisik) [inline web_hideleftmenu/__openerp__.py]: done

**Comment by aaltinisik** [comment]
Dear, @elicoidal thank you for your tutorial like review. I will make it pass travis and send another commnet.

**Comment by aaltinisik** [comment]
Dear @elicoidal, please check againif you can. I corrected all your points. @pedrobaeza I corrected it, you can check from runbot.

Regards,

**Comment by pedrobaeza** [comment]
Thanks for the update. Now it works, but it forces to click on the menu each time you change the view for hiding the menu. The ideal is to integrate the hiding option as a button in the left part itself, and that button can be toggled to hide/unhide the left menu. Do you think you can do this?

File: web_hideleftmenu/__init__.py [hunk 13]
```diff
@@ -0,0 +1,3 @@
+# -*- coding: utf-8 -*-
+# © Vauxoo <nhomar@vauxoo.com>
```
Comment (elicoidal) [inline web_hideleftmenu/__init__.py:2]: @aaltinisik Dont you want to add your copyright?

Comment (aaltinisik) [inline web_hideleftmenu/__init__.py:2]: No, I just made migration to v8.

**Comment by aaltinisik** [comment]
Dear, @pedrobaeza  I think this way of working is better. I mostly need to hide left menu to see a big tree view, or to open two fom views next to each other in two browser windows. So I need a temporary solution. Since module does not offer another way of navigation it is better this way.

I tried and used few other modules and think this is the easiest one to use.  For V7 openerp india had that module with fancy drop down menus etc. I find this one more useful and used this one. 
Also I can not do not that. Not that much developer yet :)

@hbrunn also worked on a hide menu module, that hides menu auto with mouse move. I hope it will be added to the repo also.
https://github.com/hbrunn/web/tree/8.0-web_hide_menu
Both modules will have different usage scenarios.

**Comment by pedrobaeza** [comment]
OK, I'll prefer the other approach, but I don't block this one.

Technically :+1:

**Comment by elicoidal** [comment]
:+1:

**Comment by hbrunn** [comment]
:+1:

**Comment by hbrunn** [comment]
and #131 still lingers unmerged :cry:

**Comment by pedrobaeza** [comment]
@hbrunn, but that has easy solution :wink:

**Comment by aaltinisik** [comment]
This is my first OCA merge, Thank you for your reviews, @elicoidal , @pedrobaeza, @hbrunn   I did l10n-turkey for 6.1 (it is still in odoo repo but not) Now I would like to do it again for V8, I want to do in in OCA. 
From who I can ask to open a l10n-turkey repository so I can send my PRs ?

**Comment by hbrunn** [comment]
@aaltinisik thanks for your efforts!
Send your proposal to OCA as described in https://odoo-community.org/page/suggest-a-project (and read through the site anyways, contains a lot of useful stuff)

**Pull request merged**